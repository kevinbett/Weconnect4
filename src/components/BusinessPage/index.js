import React from 'react';
import Navigation from '../Navigation/index';
import BusinessBody from '../BusinessBody';


const BusinessPage = () => (

   <div>
       <Navigation />
       <BusinessBody />
      
   </div>
);

export default BusinessPage;