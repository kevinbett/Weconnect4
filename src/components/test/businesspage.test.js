import BusinessPage from "../BusinessPage"
import { shallow } from "enzyme"

describe('Business page',() => { 
    it ('renders business page', () => { 
        expect(shallow(<BusinessPage/>)).toMatchSnapshot()
    });
} )