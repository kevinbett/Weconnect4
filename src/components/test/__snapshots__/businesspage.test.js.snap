// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business page renders business page 1`] = `
<div>
  <Navigation />
  <RenderBusiness />
</div>
`;
